tools/run_tests/performance/scenario_config.py

```diff
@@ -456,6 +456,128 @@ def scenarios(self):
   def __str__(self):
     return 'node'
 
+class PythonLanguage:
+
+  def __init__(self):
+    self.safename = 'python'
+
+  def worker_cmdline(self):
+    return ['tools/run_tests/performance/run_worker_python.sh']
+
+  def worker_port_offset(self):
+    return 500
+
+  def scenarios(self):
+    yield {
+        'name': 'python_to_cpp_protobuf_streaming_ping_pong_secure',
```

Comment:
FYI, I've removed the _secure suffix for other wrapped languages, but it's totally fine to leave it here I think.